ratauth-spring-boot-autoconfigure/src/main/groovy/ru/ratauth/server/local/PersistenceServiceStubConfiguration.groovy

```diff
@@ -97,18 +99,18 @@ class PersistenceServiceStubConfiguration {
       Observable<SessionClient> getSessionClient(String name) {
         if (name == CLIENT_NAME)
           return Observable.just(new SessionClient(
-            id: 'id',
-            name: CLIENT_NAME,
-            secret: CLIENT_SECRET,
-            salt: SALT,
-            password: SecurityUtils.hashPassword(PASSWORD, SALT),
+              id: 'id',
```

Comment:
Me and Kirill were using two spaces. Maybe the rules was changed. Ask Kirill about it.